tests/ops/qubit/test_parametric_ops.py

```diff
@@ -710,20 +709,42 @@ def test_phase_shift(self, tol):
         assert np.allclose(qml.PhaseShift.compute_matrix(phi), expected, atol=tol, rtol=0)
         assert np.allclose(qml.U1.compute_matrix(phi), expected, atol=tol, rtol=0)
 
-    def test_global_phase(self, tol):
+    @pytest.mark.parametrize("n_wires", [0, 1, 2])
+    def test_global_phase(self, tol, n_wires):
```

Comment:
Maybe parametrize `phi` here to test broadcasted dimensions for matrix as well?

Reply:
You mean instead of running broadcasted and non-broadcasted within one test? I followed the scheme in other tests, but no objection to changing it :)

Reply:
Is `GlobalPhase.matrix` tested with broadcasted dimensions anywhere currently?

Reply:
Oh sorry I misread, my bad